packages/caliper-fabric/lib/adaptor-versions/v2/fabric-gateway-v2.js

```diff
@@ -1692,41 +579,20 @@ class Fabric extends BlockchainInterface {
      * @async
      */
     async init(workerInit = false) {
-        let tlsInfo = this.networkUtil.isMutualTlsEnabled() ? 'mutual'
+        const tlsInfo = this.networkUtil.isMutualTlsEnabled() ? 'mutual'
             : (this.networkUtil.isTlsEnabled() ? 'server' : 'none');
-        let compMode = this.networkUtil.isInCompatibilityMode() ? '; Fabric v1.0 compatibility mode' : '';
-        logger.info(`Fabric SDK version: ${this.version.toString()}; TLS: ${tlsInfo}${compMode}`);
+        logger.info(`Fabric SDK version: ${this.version.toString()}; TLS: ${tlsInfo}`);
 
+        logger.warn(`Administrative actions are not possible with Fabric SDK version: ${this.version.toString()}`);
         await this._initAdaptor(workerInit);
```

Comment:
Passing `workerInit` is pointless, since it's missing from the signature of `_initAdaptor()`

Reply:
agree with that one, will remove